File: Examples/CSharp/Presentations/AccessModifyingProperties.cs
using System.IO;
using Aspose.Slides;

namespace Aspose.Slides.Examples.CSharp.Presentations
{
    public class AccessModifyingProperties
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Presentations();

            //Instanciate the Presentation class that represents the PPTX
            Presentation presentation = new Presentation(dataDir + "AccessModifyingProperties.pptx");

            //Create a reference to DocumentProperties object associated with Prsentation
            IDocumentProperties documentProperties = presentation.DocumentProperties;

            //Access and modify custom properties
            for (int i = 0; i < documentProperties.CountOfCustomProperties; i++)
            {
                //Display names and values of custom properties
                System.Console.WriteLine("Custom Property Name : " + documentProperties.GetCustomPropertyName(i));
                System.Console.WriteLine("Custom Property Value : " + documentProperties[documentProperties.GetCustomPropertyName(i)]);

                //Modify values of custom properties
                documentProperties[documentProperties.GetCustomPropertyName(i)] = "New Value " + (i + 1);
            }

            //Save your presentation to a file
            presentation.Save(dataDir + "CustomDemoModified.pptx", Aspose.Slides.Export.SaveFormat.Pptx);
        }
    }
}

File: Examples/VisualBasic/Presentations/AccessModifyingProperties.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Slides
Imports Aspose.Slides.Export

Namespace Aspose.Slides.Examples.VisualBasic.Presentations
    Public Class AccessModifyingProperties
        Public Shared Sub Run()

            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Presentations()

            ' Instanciate the Presentation class that represents the PPTX
            Dim pres As New Presentation(dataDir & "AccessModifyingProperties.pptx")

            ' Create a reference to DocumentProperties object associated with Prsentation
            Dim dp As IDocumentProperties = pres.DocumentProperties

            ' Access and modify custom properties
            For i As Integer = 0 To dp.CountOfCustomProperties - 1
                ' Display names and values of custom properties
                System.Console.WriteLine("Custom Property Name : " & dp.GetCustomPropertyName(i))
                System.Console.WriteLine("Custom Property Value : " & dp.GetCustomPropertyName(i))

                ' Modify values of custom properties
                dp(dp.GetCustomPropertyName(i)) = "New Value " & (i + 1)
            Next i

            ' Save your presentation to a file
            pres.Save(dataDir & "CustomDemoModified_out.pptx", SaveFormat.Pptx)

        End Sub
    End Class
End Namespace